src/main/java/com/efficio/fieldbook/web/trial/bean/ExpDesignParameterUi.java

```diff
@@ -7,10 +7,8 @@
 
 public class ExpDesignParameterUi implements Serializable, TabInfoBean {
 
-	/**
-	 *
-	 */
 	private static final long serialVersionUID = -3081300491744812262L;
+	private List<Integer> trialInstancesForDesignGeneration;
 	private Integer noOfEnvironments;
```

Comment:
@mderamos - it would be good to remove `noOfEnvironments `and `noOfEnvironmentsToAdd`